### ISSUE TITLE
Running styled-components + sass together

### ISSUE THREAD

**Issue by GodOfGrandeur** [stylelint/stylelint#4851]
I've setup a project originally using sass with styled components. They compile and run together until we can migrate the codebase over from sass but wondering if I can keep the linting of both working. So far it doesn't seem to work and wondering if this is a bug or limitation of stylelint or am I don't something wrong. 

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "processors": [
    "stylelint-processor-styled-components"
  ],
  "extends": [
    "stylelint-config-recommended",
    "stylelint-config-styled-components"
  ]
}
```

> Which version of stylelint are you using?

`13.2.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

"CLI with `npx stylelint 'src/**/*.{scss,js}'`"

> What did you expect to happen?

Both the scss and styled-components to be linted together.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
src/styles/base.scss
 1:1  ✖  Unexpected keyword 'import' (1:1) 
```

**Comment by jeddy3** [comment]
Try removing the styled-components processor to rely on stylelint built-in syntax switching:

```json
{
  "extends": [
    "stylelint-config-recommended",
    "stylelint-config-styled-components"
  ]
}
```

**Comment by GodOfGrandeur** [comment]
This worked, thanks.
